ember_debug/utils/ember/object/internals.js

```diff
@@ -1,14 +1,9 @@
 import Ember, { ObjectInternals } from '../../ember';
 
-let module;
-
-if (ObjectInternals) {
-  module = ObjectInternals;
-} else {
-  module = Ember;
-}
-
-let { cacheFor, guidFor: emberGuidFor } = module;
```

Comment:
I encountered a situation with 3.24 where `ObjectInternals` was defined but didn't contain a definition for `guidFor` and `cacheFor`, which makes this logic break. That's why I refactored.